### DISCUSSION TITLE
Debian 11 chrootless not taking changes

### DISCUSSION THREAD

**Discussion by MacMilanzi** [ltsp/ltsp#753] [Q&A]
Hello, i am running debian 11 and installed LTSP. I am using Chrootless as its easier for me to just maintain one image. I have created several users on the server that when i boot the clients they automatically login. But I am having an issue with changes. When i make changes on the server, with each user (example :- changing background image), and boot the clients, this change is not reflecting. Of course i run : ltsp initrd, after the changes.  Is there something i am doing wrong? I didn't have this issue when I used Ubuntu last time.

**Comment by alkisg** [comment]
Hi @MacMilanzi, I'll convert this issue into a discussion as it's not about a bug in LTSP.
Please use this link next time:
https://github.com/ltsp/ltsp/discussions

**Comment by alkisg** [comment]
After testing via IRC, it appears to work fine now, it might have been a problem with improper logout where dconf was still in use.

**Reply by MacMilanzi** [reply]
Thank you for your help, Its working perfectly fine now after properly logging out and making sure dconf is not in use.

**Comment by MacMilanzi** [comment]
Hi, so after the solution above worked I went ahead and did auto login using the MAC addresses of the clients. But when I did that, the issue above emerged again. Is there something am doing wrong in the auto login in the ltsp.conf file? example of my auto login is 

[MAC address]
AUTOLOGIN="username"
PASSWORD_="username/password"
RELOGIN=1

**Comment by alkisg** [comment]
Αfter autologin, what's the output of this command on the client?
`mount | grep home`

**Reply by MacMilanzi** [reply]
When i run this command on the client after autologin its shows nothing, blank.
